pkg/operator/ceph/operator.go

```diff
@@ -148,7 +148,9 @@ func (o *Operator) Run() error {
 	go o.startManager(stopChan)
 
 	// watch for changes to the rook clusters
-	o.clusterController.StartWatch(namespaceToWatch, stopChan)
+	if err := o.clusterController.StartWatch(namespaceToWatch, stopChan); err != nil {
```

Comment:
Let's change all the StartWatch() methods to not return an error (and I'll stop commenting on all the instances)